pkg/kubelet/util/pod_startup_latency_tracker.go

```diff
@@ -99,7 +102,8 @@ func (p *basicPodStartupLatencyTracker) ObservedPodOnWatch(pod *v1.Pod, when tim
 	if hasPodStartedSLO(pod) {
 		podStartingDuration := when.Sub(pod.CreationTimestamp.Time)
 		imagePullingDuration := state.lastFinishedPulling.Sub(state.firstStartedPulling)
-		podStartSLOduration := (podStartingDuration - imagePullingDuration).Seconds()
+		initContainerDuration := state.containersRunningTime.Sub(state.initContainerStartTime)
+		podStartSLOduration := (podStartingDuration - imagePullingDuration - initContainerDuration).Seconds()
```

Comment:
Is it possible for init container execution to overlap image pulling?
e.g. the init container images are tiny, but the main containers are huge.  The init containers pull quickly and start running while the main containers are still pulling?
I've no idea if this is possible, but I think this calculation relies on it being impossible, otherwise you'll double count?